src/stack_in_golang.go

```diff
@@ -0,0 +1,88 @@
+package main
+import "fmt"
+type Stack interface{
```

Comment:
comment needed to explain the interface

Reply:
> comment needed to explain the interface
The interface is an explanation of the functionalities that any type which implements this holds. 
Make it simple, but no simpler.

Reply:
Add description of program as comments

Reply:
Added comments.